pkg/scripts/render.go

```diff
@@ -54,9 +54,12 @@ var containerRuntimeTemplates = map[string]string{
 			{{ if .CONFIGURE_REPOSITORIES }}
 			sudo apt-get update
 			sudo apt-get install -y apt-transport-https ca-certificates curl software-properties-common lsb-release
-			curl -fsSL https://download.docker.com/linux/$(lsb_release -si | tr '[:upper:]' '[:lower:]')/gpg |
-				sudo apt-key add -
-			sudo add-apt-repository "deb https://download.docker.com/linux/$(lsb_release -si | tr '[:upper:]' '[:lower:]') $(lsb_release -cs) stable"
```

Comment:
Why did we replace `$(lsb_release -si | tr '[:upper:]' '[:lower:]')` with `ubuntu`? That was a little neat trick that allowed us to use KubeOne with both Ubuntu and Debian, even though Debian isn't officially supported.

Reply:
It took it from our Ubuntu OSP.

Reply:
Created https://github.com/kubermatic/kubeone/pull/3508 and https://github.com/kubermatic/operating-system-manager/pull/431 to address this comment